resource/writers/match_writers.cpp

```diff
@@ -148,6 +148,59 @@ int sim_match_writers_t::emit_vtx (const std::string &prefix,
     return 0;
 }
 
+/****************************************************************************
+ *                                                                          *
+ *              K8S Writers Class Public Method Definitions              *
+ *                                                                          *
+ ****************************************************************************/
+
+
+int k8s_match_writers_t::emit (std::stringstream &out)
+{
+    out  << m_out.str ();
+    return 0;
+}
+
+
+bool k8s_match_writers_t::empty ()
+{
+    m_out.str ("");
+    m_out.clear ();
+    return true;
+}
+
+int k8s_match_writers_t::emit_json (json_t **j_o, json_t **aux)
+{
+    json_t *o = NULL;
+    std::string str = m_out.str ();
+
+    if (!str.empty ()) {
+        if ( !(o = json_string (str.c_str ()))) {
+            errno = ENOMEM;
+            return -1;
+        }
+    }
+    *j_o = o;
+    return 0;
+}
+
+
+int k8s_match_writers_t::emit_vtx (const std::string &prefix,
+                                    const f_resource_graph_t &g, const vtx_t &u,
+                                    unsigned int needs, bool exclusive)
+{
+    std::string mode = (exclusive)? "x" : "s";
+    m_out <<  prefix << g[u].name << "[" << needs << ":" << mode  << "] " << std::endl;
+    if (g[u].type == "node")
+        nodename = g[u].name;
```

Comment:
Fluxion designates member variables with `m_`:
```suggestion
        m_nodename = g[u].name;
```